deploy/operator.yaml

```diff
@@ -61,3 +61,7 @@ spec:
                   fieldPath: metadata.name
             - name: OPERATOR_NAME
               value: "ibm-management-ingress-operator"
+            - name: POD_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
```

Comment:
Can you have dummy env for route https/http port, version, cluster name too in the operator deployment?

And you also need to update CSV of operator.